oauth2client/contrib/gce.py

```diff
@@ -36,10 +35,7 @@
 logger = logging.getLogger(__name__)
 
 # URI Template for the endpoint that returns access_tokens.
-_METADATA_ROOT = ('http://metadata.google.internal/computeMetadata/v1/'
-                  'instance/service-accounts/default/')
-META = _METADATA_ROOT + 'token'
-_DEFAULT_EMAIL_METADATA = _METADATA_ROOT + 'email'
+_METADATA_ROOT = 'http://metadata.google.internal/0.1/meta-data'
```

Comment:
Yeah, I was using it because I didn't think the new endpoint supported some of the info I needed, but it turns out there was a magic query arg to make it give me scopes etc. Will update.